pennylane/transforms/zx/converter.py

```diff
@@ -0,0 +1,577 @@
+                qml.T(wires=[4]),
+                qml.CNOT(wires=[2, 4]),
+                qml.adjoint(qml.T(wires=[4])),
+                qml.CNOT(wires=[1, 4]),
+                qml.T(wires=[4]),
+                qml.T(wires=[2]),
+                qml.CNOT(wires=[1, 2]),
+                qml.T(wires=[1]),
+                qml.adjoint(qml.T(wires=[2]))
+                qml.CNOT(wires=[1, 2]),
+                qml.Hadamard(wires=[4]),
+                qml.CNOT(wires=[2, 4]),
+                qml.Hadamard(wires=4),
+                qml.CNOT(wires=[1, 4]),
+                qml.adjoint(qml.T(wires=[4])),
+                qml.CNOT(wires=[0, 4]),
+                qml.T(wires=[4]),
+                qml.CNOT(wires=[1, 4]),
+                qml.adjoint(qml.T(wires=[4])),
+                qml.CNOT(wires=[0, 4]),
+                qml.T(wires=[4]),
+                qml.T(wires=[1]),
+                qml.CNOT(wires=[0, 1]),
+                qml.T(wires=[0]),
+                qml.adjoint(qml.T(wires=[1])),
+                qml.CNOT(wires=[0, 1]),
+                qml.Hadamard(wires=[4]),
+                qml.CNOT(wires=[1, 4]),
+                qml.CNOT(wires=[0, 4]),
+                return qml.expval(qml.PauliZ(wires=0))
+
+        The circuit contains 63 gates; 28 :func:`qml.T` gates, 28 :func:`qml.CNOT`, 6 :func:`qml.Hadmard` and
+        1 :func:`qml.PauliX`. We applied the `qml.transforms.to_zx` decorator in order to transform our circuit to
+        a ZX graph.
+
+        You can get the PyZX graph by simply calling the QNode:
+
+        >>> g = mod_5_4()
+        >>> pyzx.tcount(g)
+        28
+
+        PyZX gives multiple options for optimizing ZX graphs (:func:`pyzx.full_reduce`, :func:`pyzx.teleport_reduce`, ...).
+        The :func:`pyzx.full_reduce` applies all optimizations passes but it is almost certain that the reduced graph at
+        the end will be no circuit-like and it is necessary to extract the circuit. Therefore we will use :func:`pyzx.teleport_reduce`
+        which preserves the circuit structure of the graph.
+
+        >>> g = pyzx.simplify.teleport_reduce(g)
+        >>> pyzx.tcount(g)
+        8
+
+        If you give a closer look, the circuit contains now 53 gates; 8 :func:`qml.T` gates, 28 :func:`qml.CNOT`, 6 :func:`qml.Hadmard` and
+        1 :func:`qml.PauliX` and 10 :func:`qml.PauliX`. We successfully reduced the T-count by 20 and have ten additional
+        S gates and also notice that the number of CNOT remains the same.
+
+        It is possible now possible to use the optimized circuit in PennyLane:
+
+        >>> qscript =
+
+        .. code-block:: python
+
+            qscript_opt = qml.transforms.from_zx(g)
+
+            wires = qml.wires.Wires([4, 3, 0, 2, 1])
+            wires_map = OrderedDict(zip(qscript.wires, wires))
+            qscript_opt_reorder = qml.map_wires(input=qscript_opt, wire_map=wires_map)
+
+            @qml.qnode(device=dev)
+            def mod_5_4():
```

Comment:
```suggestion
            def mod_5_4_opt():
```
assuming because that's what you call below

Reply:
Correct I changed it!